README.md

```diff
@@ -88,8 +88,8 @@ your package manager):
     ENV TINI_VERSION v0.18.0
     ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
     ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /tini.asc
-    RUN gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 \
-     && gpg --verify /tini.asc
+    RUN gpg --no-tty --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 \
+     && gpg --no-tty --verify /tini.asc
```

Comment:
See the discussion over in https://bugs.debian.org/913614: TLDR is that this should be `--batch`, not `--no-tty`.